.github/workflows/rust-lint.yml

```
@@ -11,6 +11,8 @@ jobs:
     runs-on: ubuntu-latest
     steps:
       - uses: actions/checkout@v2
+      - run: mkdir -p $HOME/.cargo
+      - run: cp .github/workflows/.cargo/* $HOME/.cargo
```

There are a lot of files read from this directory. I am not sure a glob star is the best.

Lets be explicit so we dont have surprises.

```suggestion
      - run: cp .github/workflows/.cargo/audit.toml $HOME/.cargo
```